README.md

```diff
@@ -35,6 +35,15 @@ Create a bookmark with `javascript: (function () { window.location.href = 'https
 
 ![screenshot](img/screenshot.png)
 
+## Browser Support
+
+|Platform/Browser|![chrome](img/icons/chrome.png)|![safari](img/icons/safari.png)|![firefox](img/icons/firefox.png)|![edge](img/icons/edge.png)|![ie](img/icons/ie.png)|
+|:---:|:---:|:---:|:---:|:---:|:---:|
+|![windows](img/icons/windows.png)|Yes|-|Yes|Yes|Yes|
+|![mac](img/icons/mac.png)|Yes|No|Yes|-|-|
+|![ios](img/icons/ios.png)|-|No|-|-|-|
+|![android](img/icons/android.png)|No|-|No|-|-|
```

Comment:
Don't change it, but I think Chrome on Android actually works without autoplay 😆